src/main/java/edu/agh/wfiis/designpatterns/patternsUsageExample/DesignPattern.java

```diff
@@ -1,6 +1,6 @@
 package edu.agh.wfiis.designpatterns.patternsUsageExample;
 
-class DesignPattern {
+public class DesignPattern {
 
     private String id;
```

Comment:
Pola w DesignPatern mogłyby być final, skoro i tak nie ma setterów i wszystkie są ustawiane w Builderze, tylko to raczej wymaga lekkiej zmiany w konstruktorze DesignPattern